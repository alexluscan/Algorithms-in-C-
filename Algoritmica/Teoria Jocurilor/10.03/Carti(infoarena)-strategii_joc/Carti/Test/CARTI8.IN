15
12 1
8 3 4 7 K Q 9 J 6 10 5 A
11 1
Q K 3 5 4 J A 8 6 9 2
13 1
8 10 5 3 J A K Q 7 2 4 9 6
5 1
8 4 Q 9 5
9 1
10 8 J 4 6 9 A K Q
13 1
J 7 8 9 2 Q A 6 3 10 K 4 5
12 1
3 Q J 6 4 10 5 7 A 9 K 2
9 1
Q K 7 10 A 4 3 2 9
6 1
4 A 7 Q 9 10
8 1
5 2 9 4 A K 6 3
8 1
J A 8 9 Q 2 K 4
7 1
8 4 J 3 6 5 Q
13 1
5 6 4 2 3 K A 7 9 Q J 8 10
3 1
A 10 8
8 1
10 Q 3 K 6 2 8 J
