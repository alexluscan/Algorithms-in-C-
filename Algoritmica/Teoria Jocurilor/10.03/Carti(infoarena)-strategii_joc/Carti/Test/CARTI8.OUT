Bob
Alice
Alice
Alice
Alice
Alice
Bob
Alice
Bob
Bob
Bob
Alice
Alice
Alice
Bob
